orchestration/prepare.go

```diff
@@ -6,22 +6,45 @@ package orchestration
 
 import (
 	"context"
+	"fmt"
+	"net"
 
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
 	"namespacelabs.dev/foundation/internal/compute"
+	"namespacelabs.dev/foundation/internal/console"
+	"namespacelabs.dev/foundation/internal/fnapi"
 	"namespacelabs.dev/foundation/internal/planning"
 	"namespacelabs.dev/foundation/internal/planning/deploy"
 	"namespacelabs.dev/foundation/internal/runtime"
+	"namespacelabs.dev/foundation/orchestration/proto"
 	"namespacelabs.dev/foundation/schema"
+	runtimepb "namespacelabs.dev/foundation/schema/runtime"
 	"namespacelabs.dev/foundation/std/cfg"
 	"namespacelabs.dev/foundation/std/execution"
 	"namespacelabs.dev/foundation/std/tasks"
 )
 
-const orchestratorStateKey = "foundation.orchestration"
+const (
+	orchestratorStateKey                    = "foundation.orchestration"
+	serverId                                = "0fomj22adbua2u0ug3og"
+	serverName                              = "orchestration-api-server"
+	serverPkg            schema.PackageName = "namespacelabs.dev/foundation/orchestration/server"
+	toolPkg              schema.PackageName = "namespacelabs.dev/foundation/orchestration/server/tool"
+)
 
 var (
 	UseOrchestrator              = true
+	UsePinnedOrchestrator        = true
 	RenderOrchestratorDeployment = false
+	ForceOrchestratorDeployment  = false
```

Comment:
Am getting a bit lost in all of these permutations. :-D

useOrchestrator seems important
lets collapse usepinned and force? e.g. "--use_head_orchestrator", which both skips version check and unconditionally deploys the latest built on.